lib/inc/whereami/result.hpp

```diff
@@ -69,7 +75,7 @@ namespace whereami { namespace detectors {
         /**
          * Metadata about the hypervisor
          */
-        metadata metadata_;
+        class metadata metadata_;
```

Comment:
This `class` is needed to disambiguate the `metadata` method above, right?

Reply:
That's right